Homework-Feb-29/BWT/SuffixArray.cs

```diff
@@ -0,0 +1,44 @@
+namespace BWT;
+
+static class SuffixArray
+{
+    public static List<int> BuildSuffixArray(string sourceString)
+    {
+        var suffixArray = new List<int>();
+        for (var i = 0; i < sourceString.Length; ++i)
+        {
+            suffixArray.Add(i);
+        }
+        SortSuffixArray(sourceString, suffixArray);
+        return suffixArray;
+    }
+
+    private static int CompareSuffix(string sourceString, 
+        int suffixStartPos1, int suffixStartPos2)
```

Comment:
Если хотите переносить параметры на новую строку, то должно выглядеть так:
```suggestion
    private static int CompareSuffix(
        string sourceString, 
        int suffixStartPos1, 
        int suffixStartPos2)
```

Но в данном случае всего три параметра, не нужно переносить на новую строку